packages/main/src/Table.js

```diff
@@ -433,13 +434,15 @@ class Table extends UI5Element {
 	onBeforeRendering() {
 		const columnSettings = this.getColumnPropagationSettings();
 		const columnSettingsString = JSON.stringify(columnSettings);
+		const rowsCount = this.rows.length;
 
-		this.rows.forEach(row => {
+		this.rows.forEach((row, index) => {
 			if (row._columnsInfoString !== columnSettingsString) {
 				row._columnsInfo = columnSettings;
 				row._columnsInfoString = JSON.stringify(row._columnsInfo);
 			}
 
+			row._ariaPosition = this.i18nBundle.getText(TABLE_ROW_POSITION, index + 1, rowsCount);
```

Comment:
Don't you need to pass the id of the group row as we discussed?

Reply:
We discussed this with Elena and it turned out that we don't need it as it will not work anyways. That is why I have switched back the implementation to using "td" element istead of "th"